### PR TITLE
add Scala 3 support

### PR DESCRIPTION
**Pull request by maxerothan**

### CODE REVIEW

File: cluster/src/test/scala/com/evolutiongaming/cluster/AddressHelperSpec.scala [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+import org.scalatest.matchers.should.Matchers
 
-class AddressHelperSpec extends FunSuite with Matchers {
+class AddressHelperSpec extends AnyFunSpec with Matchers {
   val global = Address("akka.tcp", "coreservices", "127.0.0.1", 9196)
   val local = Address("akka", "coreservices")
   val addressHelper = new AddressHelper(global)
 
-  test("toLocal") {
+  it("toLocal") {
```
Comment (mr-git) [inline cluster/src/test/scala/com/evolutiongaming/cluster/AddressHelperSpec.scala:12]: what is `it`?

Comment (maxerothan) [reply cluster/src/test/scala/com/evolutiongaming/cluster/AddressHelperSpec.scala:12]: the most basic way to define test case. `test` word disappeared from API

Comment (mr-git) [reply cluster/src/test/scala/com/evolutiongaming/cluster/AddressHelperSpec.scala:12]: I will check logs, if those test cases are run...

on similar project locally I do not see such funtion

Comment (mr-git) [reply cluster/src/test/scala/com/evolutiongaming/cluster/AddressHelperSpec.scala:12]: it was ran

**Pull request merged**